sqlite/src/resolve.c

```diff
@@ -415,11 +415,11 @@ static int lookupName(
     ** the vm sniffs this out when it runs OP_Rowid and executes the comdb2
     ** backend call to get the rrn+genid.
     */
-    else if( cnt==0 && cntTab==1 && sqlite3IsComdb2Rowid(zCol) ){
+    else if( cnt==0 && cntTab==1 && sqlite3IsComdb2Rowid(pMatch->pTab, zCol) ){
        cnt = 1;
```

Comment:
Upon closer examination, I think both changes here need "&& pMatch" just prior to the sqlite3IsComdb2*() calls...

Reply:
Thanks!